INIT_CHANGES.md

```diff
@@ -0,0 +1,347 @@
+component.c = component.x       # Unit checked in 1.34, invalid before
+component.d = 1 / component.x   # Unit checked in 1.34, invalid before
+```
+
+## Simulations
+
+Starting from version 1.34, a simulation's _default state_ is the model's initial state, which may contain expressions:
+```
+# Create a model where the initial value of x depends on y
+
+model = myokit.parse_model('''
+[[model]]
+c.x = 1 + c.y
+
+[c]
+dot(x) = 0.1
+y = 2
+''')
+
+sim = myokit.Simulation(model)
+for value in sim.default_state():
+    print(value)
+```
+will result in `myokit.Expression[1 + c.y]`, while 
+```
+print('Original')
+for value in sim.state():
+    print(value)
+    
+print('Modified')
+sim.set_constant('c.y', 10)
+```
+will show
+```
+What should this show? Should state=None initially, and then it gets evaluated
+on-the-fly when we call state()?
+```
+
+If we run the simulation, its state will deviate from the default state:
+```
+sim.run(10)
+
+print('New state:')
+for value in sim.state():
+    print(value)
+    
+print('The state no longer depends on c.y:')
+sim.set_constant('c.y', 10)
+```
+
+Even if we change the time back to 0:
+```
+sim.set_time(0)
+
+print('Still the same state:')
+for value in sim.state():
+    print(value)
+    
+print('And it does not depend on c.y:')
+sim.set_constant('c.y', 20)
+```
+shows
+```
+```
+
+Using `reset()` will reset a model to `t=0` and to its intial state:
+```
+sim.reset()
+
+print('Back to the initial state:')
+for value in sim.state():
+    print(value)
+    
+print('Which depends on c.y:')
+sim.set_constant('c.y', 30)
+for value in sim.state():
+    print(value)
+```
+shows
+```
+```
+
+However, if we change the default state with `set_default_state` or by using `pre()`, the connection to the model's initial state will be lost:
+```
+sim.pre(10)
+
+print('The default state no longer depends on c.y:')
+for value in sim.default_state():
+    print(value)
+```
+shows
+```
+```
+
+
+
+### Simulation
+
+The `Simulation` class has full support for variables in initial expressions.
+
+#### Simulation with sensitivities
+
+Sensitivities can only be calculated with respect to expressions that do not reference other variables (i.e. expressions for which `is_literal()` is True).
+Before 1.34, all initial values were literal, by definition, but starting in 1.34 you might get an error when trying to calculate sensitivities for non-literal initial values:
+
+```
+model = myokit.parse_model('''
+[[model]]
+c.x = 3
+c.y = c.p / 2
+
+[c]
+dot(x) = 1
+dot(y) = -1
+p = 32
+''')
+
+print('This is OK, because init(c.x) is literal:')
+sim = myokit.Simulation(model, sensitivities=(['c.x'], ['init(c.x)']))
+
+print('This is OK, because c.p is literal:')
+sim = myokit.Simulation(model, sensitivities=(['c.y'], ['c.p']))
+
+print('This is not OK, because init(c.y) is not literal:')
+sim = myokit.Simulation(model, sensitivities=(['c.y'], ['init(c.y)']))
+```
+
+### SimulationOpenCL and FiberTissueSimulation
+
+Not sure!
```

Comment:
these should behave in a similar way, I tried to keep all simulations consistant